R/test.R

```diff
@@ -57,9 +57,13 @@ test <- function(pkg = ".", filter = NULL, ...) {
   Sys.sleep(0.05)
   utils::flush.console() # Avoid misordered output in RStudio
 
+  # Be specific about the env used by testthat::test_dir() and the
+  # fact that devtools::test() is not expected to return a non-zero exit
+  # code if the tests breaks.
   env <- new.env(parent = ns_env)
+  stop_on_failure <- FALSE
 
-  testthat_args <- list(test_path, filter = filter, env = env, ... = ...)
+  testthat_args <- list(test_path, filter = filter, env = env, stop_on_failure = stop_on_failure, ... = ...)
```

Comment:
I think this can just be `testthat_args <- list(test_path, filter = filter, env = env, stop_on_failure = FALSE, ... = ...)` and I think you can remove the comment.

Reply:
ha oh ok, got it! Was trying to be consistent with how you handle `env`. I can update it to just add this argument.

Reply:
ok changed this in https://github.com/r-lib/devtools/pull/2099/commits/a215003a845974c43e8354ca9a71b5a9c295e302. I also rebased this PR to the most recent state of `master`